application/frontend/src/components/DocumentNode/DocumentNode.tsx

```diff
@@ -117,9 +134,19 @@ export const DocumentNode: FunctionComponent<DocumentNode> = ({ node, linkType,
                 </div>
                 <div>
                   <div className="accordion ui fluid styled f0">
-                    { links.map( (link, i) => 
-                        <DocumentNode node={link.document} linkType={type} hasLinktypeRelatedParent={isNestedInRelated()} key={i} />
-                      )
+                    {links.map((link, i) =>
+                      (!(currentUrlParams.has("filters")) ||
+                        !currentUrlParams.has("applyFilters") ||
+                        currentUrlParams.get("applyFilters") == "false") ||
+                        (currentUrlParams.get("applyFilters") == "true" &&
+                          currentUrlParams.has("filters") &&
+                          link.document.id &&
+                          currentUrlParams.getAll("filters").includes(link.document.id)) ?
```

Comment:
You can use a shorthand instead of the ternary here: 
 i.e.replace the ternary: `condition ? <div/> : ""` 
with a shorthand i.e. `condition && <div/>`